Command/DumpSitemapsCommand.php

```diff
@@ -55,7 +55,7 @@ protected function configure()
                 'target',
                 InputArgument::OPTIONAL,
                 'Location where to dump sitemaps. Generated urls will not be related to this folder.',
-                'web'
+                'public'
```

Comment:
\App\Kernel::MAJOR_VERSION >= 4 ? 'public' : 'web'

Reply:
I cannot agree with this, because it assume that `App\Kernel` class exists.
Outside a Symfony project (during testing for instance) it will fail.